internal/provider/resource_cluster.go

```diff
@@ -367,78 +368,85 @@ func clusterToDataModel(cluster *apigen_mgmt.Tenant, data *ClusterModel) diag.Di
 		})
 	}
 
+	specMetaObj := map[string]attr.Value{
+		"default_node_group": types.ObjectValueMust(
+			defaultNodeGroup,
+			map[string]attr.Value{
+				"cpu":     types.StringValue(cluster.Resources.Components.Meta.Cpu),
+				"memory":  types.StringValue(cluster.Resources.Components.Meta.Memory),
+				"replica": types.Int64Value(int64(cluster.Resources.Components.Meta.Replica)),
+			},
+		),
+	}
+
+	if cluster.Resources.MetaStore != nil && cluster.Resources.MetaStore.Type == apigen_mgmt.Etcd {
+		specMetaObj["etcd_meta_store"] = types.ObjectValueMust(
+			etcdMetaStoreAttrTypes,
+			map[string]attr.Value{
+				"default_node_group": types.ObjectValueMust(defaultNodeGroup, map[string]attr.Value{
+					"cpu":     types.StringValue(cluster.Resources.MetaStore.Etcd.Resource.Cpu),
+					"memory":  types.StringValue(cluster.Resources.MetaStore.Etcd.Resource.Memory),
+					"replica": types.Int64Value(int64(cluster.Resources.MetaStore.Etcd.Resource.Replica)),
+				}),
+				"etcd_config": types.StringValue(cluster.EtcdConfig),
+			},
+		)
+	} else {
+		specMetaObj["etcd_meta_store"] = types.ObjectNull(etcdMetaStoreAttrTypes)
+	}
+
+	specObj := map[string]attr.Value{
+		"risingwave_config": types.StringValue(cluster.RwConfig),
+		"compute": types.ObjectValueMust(
```

Comment:
we will support scale between standalone/distributed soonly https://github.com/risingwavelabs/risingwave-cloud/pull/9653

Should we also add standalone here?